2022/spring/Sterne-MW@5pm/philip-barven.md

```diff
@@ -0,0 +1,16 @@
+# First MD File Outlise Jupyter Notebook 🙌🏆
+
+## Random Stuff Part 1:
+
+I will now *attempt* to create a list of **random** stuff:
+> - The first thing on the list is to iterate the word ***random***
+> - Next, I can admire the blockquotes by listing its benefits:
+>> 1. It visually seperates information 
+>> 2. It is easy to implement using the ">" symbol
+>> 3. It demonstrates an understanding of markdown syntax
+
+## Random Stuff Part 2:
+
+Running out of ideas, part 2 will contain a **random** picture of a ***random*** creature.
+
+![Random Creature](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.reddit.com%2Fr%2FImaginaryMonsters%2Fcomments%2Fbnduog%2Frandom_creature_i_made_out_of_3_different_animals%2F&psig=AOvVaw3p70MHFFodNeM0vqtQUsr6&ust=1642506467084000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCOjnu9DbuPUCFQAAAAAdAAAAABAD)
```

Comment:
@Phil-Barv the image is not showing on the markdown; maybe the link is not an image link, but a reddit link